### PR TITLE
changed designs for dropdown

### PR DESCRIPTION
**Pull request by fakhruddinkw**
This is regarding : https://github.com/Real-Dev-Squad/todo-action-items/issues/151

**OLD**
old-drop-down-www

**NEW**
www-new

**RESPONSIVENESS**

https://github.com/Real-Dev-Squad/website-www/assets/85249044/9c9adfaa-5b5d-4fac-9a37-7fb867d9c21d

### CODE REVIEW

File: css/navbar.css
Comment (sahsisunny) [inline css/navbar.css:1]: Could you please create the variables for the color and convert the unit of measurement to REM if the value is less than 10PX?

**Comment by sahsisunny** [comment]
Where is the test?

File: css/navbar.css
Comment (shubhamsigdar1) [reply css/navbar.css:1]: +

**Review by shubhamsigdar1 — COMMENTED** [review]
great work @fakhruddinkw 
just worked on @sahsisunny comment
ping me I will approved

I just seen my-site, Is there is any arrow
image

please make video also

**Comment by fakhruddinkw** [comment]
> great work @fakhruddinkw just worked on @sahsisunny comment ping me I will approved
> 
> I just seen my-site, Is there is any arrow in your changes?? image
> 
> please make one video please and show how toggle of arrow is working

to add arrow, i ll have to add Font awesome as dependency in package.json, i have asked for approval, if its approved i ll add arrow @shubhamsigdar1

**Review by sahsisunny — CHANGES_REQUESTED** [review]
We have several HTML pages that require the same dropdown menu. Please create this dropdown using JavaScript and add it to each page.

**Review by sahsisunny — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Pull request merged**